code/stable/gamephysics/SRS/GamePhysics_SRS.tex

```diff
@@ -590,11 +595,9 @@ \subsubsection{Data Definitions}
               \item{$\mathbf{p}$ is the position (m)}
               \end{symbDescription}
 \\ \midrule \\
-Notes & \hyperref[assumpOT]{A: objectTy}
-        
-        \hyperref[assumpOD]{A: objectDimension}
+Notes & All bodies are assumed to be rigid (from \hyperref[assumpOT]{A: objectTy}) and two-dimensional (from \hyperref[assumpOD]{A: objectDimension}).
```

Comment:
Again, we can remove the 2D assumption.  The same equation is valid in 3D.

Reply:
We can also remove the assumption about damping.  If there is friction its effect is elsewhere.  This definition is unaffected.  (I'm not 100% sure about this definition overall, but now isn't the time to get into that.)